src/guides/v2.3/install-gde/system-requirements-tech.md

```diff
@@ -19,10 +19,12 @@ Magento is not supported on Microsoft Windows and macOS.
 
 Upgrading the Magento applications and extensions you obtain from Magento Marketplaces and other sources can require up to 2GB of RAM. If you are using a system with less than 2GB of RAM, we recommend you create a [swap file](https://support.magento.com/hc/en-us/articles/360032980432); otherwise, your upgrade might fail.
 
-## Composer (latest stable version)
+## Composer
 
 [Composer][] is required for developers who wish to contribute to the Magento 2 codebase or anyone who wishes to develop Magento extensions.
 
+Magento supports the latest 1.x version of Composer.
```

Comment:
"supports the latest" sounds too promising, like "100% works only with the latest". Other fresh 1.x versions work too, and we cannot be sure that next 1.x release won't break anything.

Reply:
Maybe _Magento does not support Composer 2.x_ would be more clear.

Reply:
@dshevtsov makes a good point and flipping it round makes it less likely someone will get confused. IE It definitely doesn't work on 2.x but we're not making any specific statements on 1.x 😄

Reply:
So what do you suggest? Magento supports Composer 1.9.x?

> Maybe Magento does not support Composer 2.x would be more clear.

We should document **supported** versions of third party software, not **unsupported** versions.

Reply:
I'm not sure about the list of supported versions. It would require to test each current and next 1.x versions. At the moment, a Composer dependency states `composer/composer ^1.9`, which is 1.9.x or greater but less than 2.0.0 (https://github.com/magento/magento2/network/dependencies).

Reply:
There is a way to get that version from the composer data that we use on devdocs.

Example for 2.4:

```
<!-- The 'packages' variable contains the 'packages' node of the '_data/codebase/v2_4/open-source/composer_lock.json' file
{% assign packages = site.data.codebase.v2_4.open-source.composer_lock.packages %} -->

<!-- The 'product' variable contains data of the 'magento/product-community-edition' package {% assign product = packages | where_exp: "package", "package.name == 'magento/product-community-edition'" | first %} -->

{{ product.require['composer/composer'] }}
```